pkg/plugin/state.go

```diff
@@ -630,13 +634,24 @@ func (e *AutoscaleEnforcer) reserveResources(
 
 	e.state.lock.Lock()
 	defer e.state.lock.Unlock()
+	podName := util.GetNamespacedName(pod)
 
 	// If the pod already exists, nothing to do
-	if _, ok := e.state.pods[util.GetNamespacedName(pod)]; ok {
+	_, isPodInState := e.state.pods[podName]
+	if isPodInState {
```

Comment:
These diffs are now unnecessary, right?

Reply:
They actually are, I use the `isPodInState` in the if branch right after this one original